exist-core/src/main/java/org/exist/xquery/functions/fn/FnTransform.java

```diff
@@ -396,7 +447,7 @@ private float staxExtractXsltVersion(final Source xsltStylesheet) throws XPathEx
     private static final Option<Item> INITIAL_MATCH_SELECTION = new Option<>(
             Type.ITEM,"initial-match-selection", v3_0);
     private static final Option<QNameValue> INITIAL_MODE = new Option<>(
-            Type.QNAME,"initial-match-selection", v1_0, v2_0, v3_0);
+            Type.QNAME,"initial-mode", v1_0, v2_0, v3_0);
```

Comment:
there are many quality reports on ordering; is it possible to correct them? 
![image](https://user-images.githubusercontent.com/1700062/171145167-5a4f6efd-45c8-4dda-aa32-6d00209364b6.png)